### PR TITLE
fix(table): avoid cropping the checkbox column

### PR DESCRIPTION
**Pull request by LucyChyzhova**
fix: https://github.com/Lundalogik/crm-feature/issues/2648
## Review:
- [ ] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [ ] Commits have the correct *type* for the changes made
- [ ] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

**Review by anderssonjohan — DISMISSED** [review]

File: src/components/checkbox/partial-styles/_helper-text.scss [hunk 1]
```diff
@@ -2,23 +2,16 @@
...
 
 .limel-checkbox-helper-text {
     @include shared_input-select-picker.looks-like-helper-text;
     color: shared_input-select-picker.$helper-text-color;
-}
 
-.mdc-form-field {
-    &:hover,
-    &:focus,
-    &:focus-visible,
```
Comment (Kiarokh) [inline src/components/checkbox/partial-styles/_helper-text.scss:17]: good you removed this one. `focus-within` already should cover `focus-visible`

Comment (Kiarokh) [inline src/components/checkbox/partial-styles/_helper-text.scss:17]: good you removed this one. `focus-within` already should cover `focus-visible`

Comment (Kiarokh) [inline src/components/checkbox/partial-styles/_helper-text.scss:17]: good you removed this one. `focus-within` already should cover `focus-visible`

**Comment by Kiarokh** [comment]
@anderssonjohan can you please approve this PR?
It's approved by me, but I can't do it myself for some reasons. I just see this 👇 
image

**Comment by adrianschmidt** [comment]
Testing to see if I have the same problem as Kia…

**Review by adrianschmidt — CHANGES_REQUESTED** [review]
Testing to see if I have the same problem as Kia…

**Review by jgroth — APPROVED** [review]

**Pull request merged**